fetch_gazebo/launch/include/fetch.launch.xml

```diff
@@ -31,9 +34,11 @@
     <param name="lazy" type="bool" value="True"/>
   </node>
 
-  <!-- Start a mux between application and teleop -->
```

Comment:
I'm not particularly familiar with the use of mux'ing different command inputs; can you describe why this block is/should be removed? Thanks!

Reply:
Thank you for your review and sorry for my lack of explanation.

This mux node is actually not removed. This is included in `teleop.launch.xml`.
https://github.com/fetchrobotics/fetch_robots/blob/5fd6302bd17175c11d0ddfb4171f8513bc82a5b5/fetch_bringup/launch/include/teleop.launch.xml#L27-L29